### PR TITLE
Added a service to the File Transfer

### PR DESCRIPTION
**Pull request by AlixPlayzz**
Added a service to the File Transfer Peer-to-peer Filesharing category.
"- [mork](https://github.com/dojje/mork) - Simple command line based file sending utility, written in rust. ([Source Code](https://github.com/dojje/mork)) `GPL-3.0` `Rust`"

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software)
  must be added to `non-free.md` and marked `⊘ Proprietary`:
  ``- [Name](http://homepage/) `⊘ Proprietary` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -879,6 +879,7 @@ _Related: [Groupware](#groupware)_
 - [Send](https://github.com/timvisee/send) - Simple, private, end to end encrypted temporary file sharing, originally built by Mozilla. ([Clients](https://github.com/timvisee/send#clients)) `MPL-2.0` `Nodejs`
 - [Torrents.csv](https://codeberg.org/heretic/torrents-csv-server) - A self-hostable torrent search engine. `GPL-3.0` `Rust`
 - [Transmission](https://transmissionbt.com/) - Fast, easy, Free Bittorrent client. ([Source Code](https://github.com/transmission/transmission)) `GPL-3.0` `C`
+- [mork](https://github.com/dojje/mork) - Simple command line based file sending utility, written in rust. ([Source Code](https://github.com/dojje/mork)) `GPL-3.0` `Rust`
```
Comment (bertiebaggio) [inline README.md:882]: Do not add a duplicate `Source code` link if it is the same as the main link :)

**Review by bertiebaggio — CHANGES_REQUESTED** [review]

**Review by AlixPlayzz — COMMENTED** [review]
fixed source code and alphabetical order

File: README.md (hunk 1 above)
Comment (AlixPlayzz) [reply README.md]: Oh right sorry, that makes alot of sense. First pull request ever, so not very good at this lmao. I believe I fixed it now.

Comment (bertiebaggio) [reply README.md]: No worries, that's what the reviews are for :)

File: README.md [hunk 2]
```diff
@@ -872,6 +872,7 @@ _Related: [Groupware](#groupware)_
 - [instant.io](https://github.com/webtorrent/instant.io) - Streaming file transfer over WebTorrent. ([Demo](https://instant.io)) `MIT` `Nodejs`
 - [Magnetico](https://github.com/boramalper/magnetico) - Magnetico is the first autonomous (self-hosted) BitTorrent DHT search engine suite that is designed for end-users. `AGPL-3.0` `Python`
 - [Magnetissimo](https://github.com/sergiotapia/magnetissimo) - Search engine that indexes all popular torrent sites. `MIT` `Elixir`
+- [mork](https://github.com/dojje/mork) - Simple command line based file sending utility, written in rust. `GPL-3.0` `Rust`
```
Comment (nodiscc) [inline README.md:875]: > the program will contact the tracker server which keeps track of all the codes

I understand this repository contains source code for both the command-line client and the server? How do I run the server?

Comment (nodiscc) [reply README.md:875]: ping @AlixPlayzz

**Comment by nodiscc** [comment]
No reply for > a month, closing, feel free to reopen

In addition the project's README.md states:

> mork (NOT USABLE)

**Pull request closed without merging**